MODULE.bazel

```diff
@@ -0,0 +1,52 @@
+module(
+    name = "rules_oci",
+    repo_name = "com_github_datadog_rules_oci",
+)
+
+bazel_dep(name = "aspect_bazel_lib", version = "2.7.3")
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
+bazel_dep(name = "bazel_skylib_gazelle_plugin", dev_dependency = True, version = "1.6.1")
+bazel_dep(name = "gazelle", version = "0.36.0")
+bazel_dep(name = "rules_go", version = "0.47.1")
+bazel_dep(name = "rules_pkg", version = "0.10.1")
+bazel_dep(name = "stardoc", version = "0.6.2")
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** I think this bump to `stardoc` fixes [this issue](https://github.com/DataDog/rules_oci/pull/59#discussion_r1622880120) we had with some of our documentation.